modules/Lending.py

```diff
@@ -144,32 +151,30 @@ def get_min_loan_size(currency):
 
 def create_lend_offer(currency, amt, rate):
     days = '2'
-    # if (min_daily_rate - 0.000001) < rate and Decimal(amt) > min_loan_size:
-    if float(amt) > get_min_loan_size(currency):
-        if float(rate) > 0.0001:
-            rate = float(rate) - 0.000001  # lend offer just bellow the competing one
-        amt = "%.8f" % Decimal(amt)
-        if float(rate) > xday_threshold:
-            days = xdays
-        if xday_threshold == 0:
-            days = '2'
-        if Config.has_option('BOT', 'endDate'):
-            days_remaining = int(Data.get_max_duration(end_date, "order"))
-            if int(days_remaining) <= 2:
-                print "endDate reached. Bot can no longer lend.\nExiting..."
-                log.log("The end date has almost been reached and the bot can no longer lend. Exiting.")
-                log.refreshStatus(Data.stringify_total_lent(*Data.get_total_lent()), Data.get_max_duration(
-                    end_date, "status"))
-                log.persistStatus()
-                exit(0)
-            if int(days) > days_remaining:
-                days = str(days_remaining)
-        if not dry_run:
-            msg = api.create_loan_offer(currency, amt, days, 0, rate)
-            if days == xdays and notify_conf['notify_xday_threshold']:
-                text = "{0} {1} loan placed for {2} days at a rate of {3:.4f}%".format(amt, currency, days, rate * 100)
-                log.notify(text, notify_conf)
-            log.offer(amt, currency, rate, days, msg)
+    if float(rate) > 0.0001:
```

Comment:
I removed "if float(amt) > get_min_loan_size(currency):" because it is checked already before function call, in function "lend_cur"  and in get_cur_spread, called by construct_orders)